### DISCUSSION TITLE
Cypress tests - signup failure [Nov 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4421] [Discord Threads]
> 🙋‍♂️ **genne**  
## For the cypress integration test, all tests fail because it can't sign up with preset username. Can you advise how I should avoid this? Thanks  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1033043766480732190/Screen_Shot_2022-10-21_at_11.45.43_AM.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### <@857998597474484295>  
#  
> 🙋‍♂️ **genne**  
#### FYI, I ran cypress:clear script, and reinstall cypress. But it didn't help me.  
#  
> 🙋‍♂️ **Braks**  
#### It's got nothing to do with you installing cypress wrong or anything like that   
####    
#### The Cypress tests suite expects a blank NocoDB installation - so if you already ran the tests on your noco instance, they won't run a second time on the same instance (as the user already exists)   
####    
#### You can (for local testing) change the signUp to signIn in `00_pre_configurations.js` if the user already exists   
####    
#### ```js   
#### it("Admin SignUp", () => {   
####    // change signUp to signIn   
####    loginPage.signIn(roles.owner.credentials);   
#### });   
#### ```  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1033043766837260398)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
